dash/include/dash/coarray/CoEventRef.h

```diff
@@ -5,14 +5,21 @@
 #include <dash/GlobPtr.h>
 #include <dash/Atomic.h>
 
+#include <dash/memory/MemorySpace.h>
+
 namespace dash {
 namespace coarray {
 
 class CoEventRef {
 private:
   using self_t      = CoEventRef;
   using event_ctr_t = dash::Atomic<int>;
-  using gptr_t      = GlobPtr<event_ctr_t>;
+  //TODO rko: fix this hard coded information it is related to Coevent
+  //which uses dash::Array internally
+  using globmem_t =
```

Comment:
Same here

Reply:
Hmm...not sure what you mean? The only way I see is to provide the pointer type as a template parameter to `CoEventIter`.